README.md

```diff
@@ -1,2 +1,103 @@
-# module-ballerinai-observe
-Observability components related to the internal executions in Ballerina
+
+Ballerina Observe Internal Library
+===================
+
+[![Build](https://github.com/ballerina-platform/module-ballerinai-obser/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/build-timestamped-master.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/trivy-scan.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinai-observe.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/commits/master)
+[![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/http.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fhttp)
```

Comment:
The labels filter should change here I think 🤔  We may have to add two labels for the two repos.